eng/docgeneration/templates/matthews/styles/main.js

```diff
@@ -78,24 +79,62 @@ function httpGetAsync(targetUrl, callback) {
 }
 
 function populateIndexList(selector, packageName) {
-    url = "https://azuresdkdocs.blob.core.windows.net/$web/" + SELECTED_LANGUAGE + "/" + packageName + "/versioning/versions"
-
-    httpGetAsync(url, function (responseText) {
-
-        var publishedversions = document.createElement("ul")
+    var url = "https://azuresdkdocs.blob.core.windows.net/$web/" + SELECTED_LANGUAGE + "/" + packageName + "/versioning/versions"
+    var latestGAUrl = "https://azuresdkdocs.blob.core.windows.net/$web/" + SELECTED_LANGUAGE + "/" + packageName + "/versioning/latest-ga"
+    var latestPreviewUrl = "https://azuresdkdocs.blob.core.windows.net/$web/" + SELECTED_LANGUAGE + "/" + packageName + "/versioning/latest-preview"
+    var latestVersions = document.createElement("ul")
+    httpGetAsync(latestGAUrl, function (responseText) {
         if (responseText) {
-            options = responseText.match(/[^\r\n]+/g)
-
-            for (var i in options) {
-                $(publishedversions).append('<li><a href="' + getPackageUrl(SELECTED_LANGUAGE, packageName, options[i]) + '" target="_blank">' + options[i] + '</a></li>')
-            }
+            version = responseText.match(/[^\r\n]+/g)
+            $(latestVersions).append('<li><a href="' + getPackageUrl(SELECTED_LANGUAGE, packageName, version) + '" target="_blank">' + 'Latest GA Version: ' + version + '</a></li>')
         }
         else {
-            $(publishedversions).append('<li>No discovered versions present in blob storage.</li>')
+            $(latestVersions).append('<li>No latest GA versions found.</li>')
+        }                
+    })
+    httpGetAsync(latestPreviewUrl, function (responseText) {
+        if (responseText) {
+            version = responseText.match(/[^\r\n]+/g)
+            $(latestVersions).append('<li><a href="' + getPackageUrl(SELECTED_LANGUAGE, packageName, version) + '" target="_blank">' + 'Latest Preview Version: ' + version + '</a></li>')
         }
-
-        $(selector).after(publishedversions)
+        else {
+            $(latestVersions).append('<li>No latest Preview versions found.</li>')
+        }                
     })
+    var publishedVersions = $('<ul style="display: none;"></ul>')
```

Comment:
Lets see if we can make the other versions text align with the other list bullets.

Reply:
Done.